src/components/overview-table-business-view.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import {
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+} from '@material-ui/core';
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+import { formatTimestampWithoutSecond } from './commons';
+
+const processFileStatusStyles = {
+    ERROR: {
+        backgroundColor: 'red',
+        color: 'white',
+    },
+    READY: {
+        backgroundColor: 'limegreen',
+        color: 'white',
+    },
+    RUNNING: {
+        backgroundColor: 'blue',
+        color: 'white',
+    },
+    SUCCESS: {
+        backgroundColor: 'green',
+        color: 'white',
+    },
+};
```

Comment:
Can you use the same color code as we defined in table-header.js to define the color of the status chip ?